server/models/articleModel.js

```diff
@@ -0,0 +1,28 @@
+const pool = require("./db");
+
+module.exports = () => {
+  const createCategoryTable = async () => {
```

Comment:
why is this named category?

Reply:
It was a mistake sir. I have changed it now